cmd/gaia/app/sim_test.go

```diff
@@ -125,7 +125,7 @@ func TestFullGaiaSimulation(t *testing.T) {
 		},
 		[]simulation.RandSetup{},
 		[]simulation.Invariant{
-			allInvariants,
+			simulation.PeriodicInvariant(allInvariants, 100, 0),
```

Comment:
Would it make sense to break up these invariants more? i.e. bank invariant every 100, staking/slashing every time?

Reply:
Possibly! Currently optimized for ease of changing periodicity...

Reply:
Hrmm. Maybe we should have a seperate convenient `debug` full simulation which takes in the seed and then run the invariants every time. (i.e. I'd prefer if we had a very easy way to see _exactly_ what caused that randomized error once we've found an error)

Reply:
We can search by bisection.

Reply:
(and also have a way to run the invariants every time, sure)